ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -13,15 +14,19 @@
   }}
 />
 
-<form {{on "submit" (perform this.setAssociation)}} class={{unless (or this.error this.syncedSecret) "has-top-margin-m"}}>
```

Comment:
Curious why this class was removed since it added space to match the designs? It's nicer without it given the conditional but just wondering.

Reply:
Ah looking at the designs it seemed like a smaller margin than what I saw in the UI